### PR TITLE
Eliminate old tutorial from SasView 5x

### PR DESCRIPTION
**Pull request by smk78**
This PR completely removes the old v2 tutorial (that had been kept through v3x and 4x) from the SasView repo.

The old tutorial (in all its guises) has, however, been preserved in the tutorials repo through the separate commit https://github.com/SasView/tutorials/commit/c0e852a48419c1c8f3a7c2610928882a0279e174

A copy of subtracting_a_model_calculation_from_real_data_v4x.pdf has been added as subtracting_a_model_calculation_from_real_data_v5x.pdf.

tutorials.rst has been updated as required.

### CODE REVIEW

**Review by butlerpd — CHANGES_REQUESTED** [review]
I have not yet tested the changes but the coding changes look fine save one point:

Opening the file renamed from v4 to v5 seems it seems to actually be a version 4 tutorial. Nothing has really changed I don't think in the add/subtract data but all the GUI images are clearly 4.x images.  All other files seem to be of the 5.x GUI.

Not sure the best way to address, other than the nomenclature of the file should probably stay at 4.x. Perhaps that is all that is needed since nothing but the "look" changed. There is no label for the user to see saying the graphics are from any particular version.  Alternatively I suppose we could add a parenthentical comment on that particular line saying GUI images are from 4.x but that the tutorial is otherwise uptodate?

**Comment by smk78** [comment]
Replying to @butlerpd:

Yes, well spotted. It was on my to-do list (really)! :-) Have just pushed a revised version of that tutorial updated to v5 Win.

**Comment by butlerpd** [comment]
I am confused @smk78 this was moved back to under review but nothing has changed? Do the link still does not work.  Did I miss something?

**Comment by smk78** [comment]
@butler: So if you look at the workflow history, you pointed out the file needed revising and moved this PR from Review to Revision. I committed an updated file and moved the PR back to Review. But for unspecified reasons you moved it back to Revision so I moved it back to Review! If you look at src/sas/sasview/media/subtracting_a_model_calculation_from_real_data_v5.pdf you will see this is now updated to v5.

**Comment by smk78** [comment]
Ahh. But I did forget to update tutorial.rst I see...

**Review by butlerpd — APPROVED** [review]
Nice job on the new tutorial... I realize it was probably just an image replacement but there were quite a few.  Anyway all seems well now.

File: docs/sphinx-docs/source/user/tutorial.rst [hunk 1]
```diff
@@ -7,18 +7,10 @@ Tutorials
...
-:download:`Old Tutorial <sasview/Tutorial.pdf>`
-
-:download:`Subtracting a Model Calculation from Experimental Data <sasview/subtracting_a_model_calculation_from_real_data_v4x.pdf>`
-
-The following tutorials have been written for Version 5.x
-
 :download:`Getting Started with Sasview <sasview/getting_started_with_sasview_v5.pdf>`
 
+:download:`Subtracting a Model Calculation from Experimental Data <sasview/subtracting_a_model_calculation_from_real_data_v5x.pdf>`
+
```
[the remark this answers is not in the document]
Comment (butlerpd) [reply docs/sphinx-docs/source/user/tutorial.rst]: not sure why this shows up here instead of above where It was the comment but it is now resolved

**Pull request merged**